src/actions/index.js

```diff
@@ -33,19 +33,28 @@ export const DOWNLOAD_SELECTION = 'DOWNLOAD_SELECTION'
 export const SHOW_FILE_ACTIONMENU = 'SHOW_FILE_ACTIONMENU'
 export const HIDE_FILE_ACTIONMENU = 'HIDE_FILE_ACTIONMENU'
 export const DOWNLOAD_FILE = 'DOWNLOAD_FILE'
-export const DOWNLOAD_FILE_MISSING = 'error.download_file.missing'
-export const DONWLOAD_FILE_OFFLINE = 'error.download_file.offline'
+export const DOWNLOAD_FILE_E_MISSING = 'DOWNLOAD_FILE_E_MISSING'
+export const DOWNLOAD_FILE_E_OFFLINE = 'DOWNLOAD_FILE_E_OFFLINE'
 export const OPEN_FILE_WITH = 'OPEN_FILE_WITH'
-export const DISPLAY_TOAST = 'DISPLAY_TOAST'
-export const HIDE_TOAST = 'HIDE_TOAST'
+export const OPEN_FILE_E_OFFLINE = 'OPEN_FILE_E_OFFLINE'
+export const OPEN_FILE_E_NO_APP = 'OPEN_FILE_E_NO_APP'
 export const ALERT_CLOSED = 'ALERT_CLOSED'
 
+const T_DOWNLOAD_FILE_MISSING = 'error.download_file.missing'
```

Comment:
Not sure these constants are necessary (there are already a lot of action types consts here...). What do you think about using string keys in the action creators instead of these consts?